run-perf

```diff
@@ -36,7 +37,8 @@ export MAIN_JOBEVENT_UNIQUE_SIZE=20000
 export OUTPUT_DATE_FROM=2026-02-01
 export OUTPUT_DATE_TO=2026-03-01
 
-/usr/bin/time --format='\n\nmemory %MK\ncpu %P\ntime %es\n\n' \
+TIME=`which gtime && echo 'gtime' || echo 'time'`
```

Comment:
you only need to do this once per file - the variable stays throughout the rest of the file so no need to set it multiple times